lcidlc/lcidlc.gyp

```diff
@@ -0,0 +1,125 @@
+{
+	# ----- lcidlc -----
+	'includes':
+	[
+		'../common.gypi',
+	],
+
+	'targets':
+	[
+        {
+			'target_name': 'lcidlc',
+			'type': 'executable',
+
+            'dependencies':
+			[
+				'../libfoundation/libfoundation.gyp:libFoundation',
+				'encode_support',
+			],
+
+			'include_dirs':
+			[
+				'include',
+				'src',
+			],
+
+			'sources':
+			[
+				'include/LiveCode.h',
+
+                'src/Coder.h',
+				'src/Coder.cpp',
+				'src/CString.h',
+				'src/CString.cpp',
+				'src/Error.h',
+				'src/Error.cpp',
+				'src/Interface.h',
+				'src/Interface.cpp',
+				'src/InterfaceGenerate.cpp',
+				'src/InterfacePrivate.h',
+				'src/Main.cpp',
+				'src/NativeType.h',
+				'src/NativeType.cpp',
+				'src/Parser.h',
+				'src/Parser.cpp',
+				'src/Position.h',
+				'src/Position.cpp',
+				'src/Scanner.h',
+				'src/Scanner.cpp',
+				'src/Value.h',
+				'src/Value.cpp',
+
+                '<(SHARED_INTERMEDIATE_DIR)/LCIDLC/EncodedJavaSupport.c',
+                '<(SHARED_INTERMEDIATE_DIR)/LCIDLC/EncodedSupport.c',
+			],
+			
+			'conditions':
+			[
+				[
+					# Don't compile on android, iOS or emscripten
+					'OS == "android" or OS == "ios" or OS == "emscripten"',
+					{
+						'type': 'none',
+					},
```

Comment:
Should be fixed now. Would be handy if it were standard JSON so we could run a validator on it. It seemed to work so obviously gyp isn't overly fussy...